org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/security/ISecurityProvider.java

```diff
@@ -139,69 +125,17 @@ public interface ISecurityProvider {
    * @throws ProcessingException
    *           If there is an error creating the hash. <br>
    */
-  default byte[] createPasswordHash(char[] password, byte[] salt) {
-    return createPasswordHash(password, salt, MIN_PASSWORD_HASH_ITERATIONS);
-  }
-
-  /**
-   * @deprecated use {@link #createPasswordHash(char[], byte[])}
-   * @param password
-   *          The password to create the hash for. Must not be {@code null} or empty.
-   * @param salt
```

Comment:
Has the JavaDoc be removed by purpose? Or should only the iterations be removed, resp. moved to SunSecurityProvider?